swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
No, it's much nicer to have a dedicated swagger-ui; Apiman used embed the swagger-ui, and it's a great tool to display the swagger.json document as well as to interact with our REST API. It makes things easy for ourselves, QE and the UI team.